### ISSUE TITLE
Questions in detector architecture of slow fast  for AVA dataset

### ISSUE THREAD

**Issue by malarsaravanan1991** [facebookresearch/SlowFast#156]
Hey hi,

Thanks for the work. I have a question regarding the detection architecture under AVA dataset Experiment.  I am trying to reproduce the result of Table9: AVA action detection baselines on resnet50. Can you please let me know the code for your detector with slow fast in the backbone?
Also can you please let me know how you had narrow down from larger proposal to some minimum number for the validation code ?Since your detector is based on Faster RCNN it would output atleast 300 proposals each class i am curious how you had managed to narrow down to less proposal and also defining multiple labels for the same bounding box.

Thanks in advance

**Comment by sureone** [comment]
From my understanding, the architecture have not integrated any RPN (like RCNN) inside to generate the region proposal.

**Comment by malarsaravanan1991** [comment]
Yes but in the paper you had mentioned " Our region proposals are computed
by an off-the-shelf person detector, i.e., that is not jointly
trained with the action detection models" under Experiments AVA Action detection . So how do you manage to narrow down these proposals to some number for validation. The above link to good example is not working :(

**Comment by banshee1** [comment]
> Yes but in the paper you had mentioned " Our region proposals are computed
> by an off-the-shelf person detector, i.e., that is not jointly
> trained with the action detection models" under Experiments AVA Action detection . So how do you manage to narrow down these proposals to some number for validation. The above link to good example is not working :(

Hi, here we only need the proposals of class 'person' and they have metioned how they filtered the proposals by the confidence score in the paper.
